gammapy/spectrum/sherpa_utils.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, fit):
     def _calc(self, data, model, *args, **kwargs):
         self.fit.calc_statval()
         # Sum likelihood over all observations
-        total_stat = np.sum(self.fit.statval, dtype=np.float64)
+        total_stat = np.sum([np.sum(v) for v in self.fit.statval], dtype=np.float64)
```

Comment:
This does not work if the two observations have a different number of bins, like so:
```
In [9]: array1 = np.array([1,2,3])

In [10]: array2 = np.array([1,2,3,4])

In [11]: listofarrays = [array1, array2]

In [12]: np.sum(listofarrays)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-12-e04d0a9e51c5> in <module>()
----> 1 np.sum(listofarrays)

/Users/larsm/Software/anaconda2/lib/python2.7/site-packages/numpy/core/fromnumeric.pyc in sum(a, axis, dtype, out, keepdims)
   1842         except AttributeError:
   1843             return _methods._sum(a, axis=axis, dtype=dtype,
-> 1844                                 out=out, **kwargs)
   1845         return sum(axis=axis, dtype=dtype, out=out, **kwargs)
   1846     else:

/Users/larsm/Software/anaconda2/lib/python2.7/site-packages/numpy/core/_methods.pyc in _sum(a, axis, dtype, out, keepdims)
     30 
     31 def _sum(a, axis=None, dtype=None, out=None, keepdims=False):
---> 32     return umr_sum(a, axis, dtype, out, keepdims)
     33 
     34 def _prod(a, axis=None, dtype=None, out=None, keepdims=False):

ValueError: operands could not be broadcast together with shapes (3,) (4,) 
```
In principle it could be possible to have observations with different binnings...